src/pages/Home.js

```diff
@@ -1,12 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Carousel, CarouselItem, CarouselControl, CarouselIndicators } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { useParams, NavLink } from 'react-router-dom';
+import ApartmentDelete from './ApartmentDelete';
 
 import '../styling/Home.css';
 
-const Home = () => {
-    return (
-    <>
-    <h1 className="tester">Hello World</h1>
-    </>
-    )
-}
-export default Home;
\ No newline at end of file
+const ApartmentCard = ({ apartment, id }) => (
```

Comment:
I believe (not sure so this is a little hint hint to look into) that this could be used all in Reactstrap but definitely take a look into some Reactstrap cards and see if that is applicable in place of this. If not this is a GENIUS alternative back door route here